src/framework/vst/vsterrors.h

```diff
@@ -46,9 +46,9 @@ inline Ret make_ret(Err e)
     case Err::Undefined: return Ret(retCode);
     case Err::UnknownError: return Ret(retCode);
     case Err::NoPluginModule: return Ret(retCode, trc("vst", "Could not create VstModule"));
-    case Err::NoPluginFactory: return Ret(retCode, trc("vst", "Could not get Vst Plugin Factory from file"));
-    case Err::NoPluginProvider: return Ret(retCode, trc("vst", "No VST3 Audio Module Class found"));
-    case Err::NoPluginController: return Ret(retCode, trc("vst", "No VST3 Editor Controller Class found"));
+    case Err::NoPluginFactory: return Ret(retCode, trc("vst", "Could not get VST plugin factory from file"));

```

Comment:
Shouldn't that also be "VST3"?

Reply:
They probably shouldn't be translatable at all

Reply:
Aha? Why not?

Reply:
They are not particularly useful to the user with this text. At the moment we don't even show them to the user, we only print them to the log

Reply:
I see. I could remove the translation ´decorations'